app/routes/admin/email/EmailUsersRoute.js

```diff
@@ -52,15 +52,14 @@ const mapStateToProps = (state) => {
 const mapDispatchToProps = { fetch, push };
 
 export default compose(
-  prepare(
-    (_, dispatch) =>
-      Promise.all([
-        dispatch(fetchAllWithType(GroupTypeCommittee)),
-        dispatch(fetchAllWithType(GroupTypeGrade)),
-      ]),
-    [],
-    { awaitOnSsr: false }
```

Comment:
Is the new function equal in behavior here with the empty dep array and `awaitOnSsr: false`?

Reply:
It should have equal behaiviour, `runSync: false` (which is roughly equivelant to `awaitOnSsr: false`) is now default. Empty dep array is also default in `withPreparedEffect`.

This does mean that most prepared effects have changed behaiviour from `awaitOnSsr: true` (which was previously default) to `false`, but it shouldn't matter as there are no other prepared effects deeper in the component tree.